### PR TITLE
Add Google analytics support

### PR DESCRIPTION
**Pull request by code-constructor**
It's now possible to enable Google Analytics. Just input your tracking ID in the "Site settings":

google analytics

### CODE REVIEW

**Comment by code-constructor** [comment]
@apepper ready for review

File: src/Components/GoogleAnalytics.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    return (
+      <Helmet>
+        <script async src='/google_analytics.js'></script>
+        <script async src='https://www.google-analytics.com/analytics.js'></script>
+      </Helmet>
+    );
+  }
+
+  getTrackingId() {
+    const rootPage = Scrivito.Obj.getByPath('/');
```
Comment (apepper) [inline src/Components/GoogleAnalytics.js]:
```jsx
const rootPage = Scrivito.Obj.root();
```

File: src/Components/GoogleAnalytics.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    return rootPage.get('googleAnalyticsTrackingId');
```
Comment (apepper) [inline src/Components/GoogleAnalytics.js]: Make sure, that `rootPage` does exist.

File: webpack.config.js [hunk 3]
```diff
@@ -71,14 +71,15 @@ module.exports = (env = {}) => {
...
+      google_analytics: './google_analytics.js',
       scrivito_extensions: './scrivito_extensions.js',
       'index.css': './assets/stylesheets/index.scss',
     },
     module: {
       rules: [
         {
           test: /\.js$/,
-          include: [path.join(__dirname, 'src')],
+          include: [path.join(__dirname, 'src'), path.join(__dirname, 'node_modules/autotrack')],
```
Comment (apepper) [inline webpack.config.js]: Please split this into mulitple lines. Somethink like this:

```js
include: [
  path.join(__dirname, 'src'),
  path.join(__dirname, 'node_modules/autotrack'),
],
```

File: src/Components/GoogleAnalytics.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import * as Scrivito from 'scrivito';
+import Helmet from 'react-helmet';
+
+class GoogleAnalytics extends React.Component {
+  componentDidMount() {
+    Scrivito.load(() => this.getTrackingId()).then(trackingId => {
```
Comment (apepper) [inline src/Components/GoogleAnalytics.js]: The following should also work:

```js
Scrivito.load(this.getTrackingId).then(trackingId => {
```

File: src/App.js [hunk 5]
```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import * as Scrivito from 'scrivito';
 import CurrentPageMetaData from './Components/CurrentPageMetaData';
+import GoogleAnalytics from './Components/GoogleAnalytics';
```
Comment (apepper) [inline src/App.js]: Please sort the import list by alphabet.

**Comment by apepper** [comment]
I also don't see changes to `package-lock.json`. Please include them in this PR.

**Comment by code-constructor** [comment]
@apepper ready for review again

**Pull request merged**